### ISSUE TITLE
Try to add Smart GateWay zigbee Hub 

### ISSUE THREAD

**Issue by vjmaz** [rospogrigio/localtuya#532]
## The problem
 I am trying to add a new ZigBee Hub ( its called Smart Gateway in the Smart Life App ) 
When i try to added its using localtuya integration  its states the following :

Connection to Device Succeeded but no datapoints found.  Please try Again . Create a new Issue and include Logs if problem persist .

I tried looking for logs under the following locations in HA 
Supervisor -> System then under log provider i looked at Supervisor , Core at al the choices 
also 
Configuration -> Logs   ( but nothing here ) or above location 

Am I suppose to be looking for logs some were specific sorry to ask 

I can also allow you access if you need to test or try your self or we can do a webex / remote session if you like 

I have added Tuya via cloud also only to check what it sees but no more than that .

Regards Maz 
vjmaz@hotmail.com 

## Environment

- Localtuya version:  3.2.2
- Last working localtuya version (if known and relevant): 
- Home Assistant Core version:  2021.7.4
- [] Are you using the Home Assistant Tuya Cloud component ? x
- [] Are you using the Tuya App in parallel ? x

## Steps to reproduce

1.
2. 
3.

## Configuration `configuration.yaml` or `config_flow`   N/A 

```yaml    N/A

```

## Provide Home Assistant taceback/logs  ( were do i get this from ) 

```txt

```
## Additional information

I have spare smartthings hub if you want to hack !

**Comment by vjmaz** [comment]
I was trying to add a screenshot but don't see a attachment or anything

**Comment by TheByteStuff** [comment]
I'm working through the same problem myself but no luck so far.  My Smart Gateway is connected to ethernet and a usb power adapter.  I added a Zigbee Plug just in case a Zigbee device needed to be configured for the gateway to be recognized.   Both the gateway and the outlet list via the "tuya-cli wizard" to query the Tuya cloud.

I'm currently running 2021.3.3 Home Assistant, Local Tuya 3.2.2.

**Comment by Scorpio256** [comment]
I got same error when I try add wireless button connected to this gateway. Also on gateway.

**Comment by vmartinv** [comment]
Gateway implementation is not ready yet, you could get it here and see if it works for you #318

**Comment by rodrigoasrodrigues** [comment]
Same problem with Ekaza Ethernet Gateway (powered by tuya), if I select protocol 3.3 I get this message
> Failed to authenticate with device. Verify that device id and local key are correct.

and if I select 3.1:
> An unknown error occurred. See log for details.

but I don't know where this log is supposed to be.
